benchmark/README.md

```diff
@@ -0,0 +1,6 @@
+Benchmarks here are generally in src/main and can be run with `benchmark/jmh:run`
+within SBT.
+
+See here for more details:
```

Comment:
Are there any suggested options that you can pull out from these referenced readmes?

Maybe 2-3 patterns that are most common for us in marathon?